addons/native_dialog/osx_dialog.m

```diff
@@ -509,6 +515,13 @@ - (NSMenuItem *) build_menu_item:(ALLEGRO_MENU_ITEM *)aitem
 static volatile ALLEGRO_EVENT_QUEUE *queue = NULL;
 static ALLEGRO_MUTEX *mutex;
 
+/* Declare this here to avoid getting a warning. 
+* There seems to be no better way to do this...
+*/
+@interface NSApplication(AllegroOSX)
```

Comment:
Could you move this next to the osx_clear_mouse declaration, so all declarations are in the same spot?